src/dev/create-extension.ts

```diff
@@ -150,15 +210,16 @@ const main = async (rawArgs: string[]) => {
     prettyLog.info(`Extension name: ${projectName}\n`);
 
     prettyLog.info("Getting list of changed files...", 1);
-    const changedFiles = await getChangedFilesFromFirstCommit(projectPath);
+    const changedFiles = await getDiffFilesFromFirstCommit({ projectPath });
+    const deletedFiles = await getDiffFilesFromFirstCommit({ projectPath, isDeleted: true });
 
-    if (changedFiles.length === 0) {
+    if (changedFiles.length === 0 && deletedFiles.length === 0) {
       prettyLog.warning("No changed files to copy.", 1);
```

Comment:
I think it's better to use `There are no file changes to copy.` here

Reply:
Yup makes sense updated 🙌